src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -160,6 +166,38 @@ object DepositHandler {
     }
   }
 
+  private def resolveFetchItems(bagitDir: File)(implicit id: String): Try[Unit] = Try {
+    log.debug(s"[$id] Checking fetch.txt")
```

Comment:
Come to think of it, I think the log line is okay, but you should actually **do** what it says: first check the fetch.txt and only try the downloads if all the lines are okay. So: fail fast, if part of the lines are okay, still reject the whole bag.